### ISSUE TITLE
Can we merge the `Client` and `CDCEtcdClient` types difinition in pkg/etcd

### ISSUE THREAD

**Issue by lonng** [pingcap/tiflow#5740]
### Before asking a question, make sure you have

- [X] Searched existing Stack Overflow questions
- [X] Googled your question
- [X] Searched open and closed [GitHub issues](https://github.com/pingcap/tiflow/issues?q=is%3Aissue)
- [X] Read the [documentation](https://docs.pingcap.com/tidb/stable)

### What is your question?

Ref: https://github.com/pingcap/tiflow/blob/18e5c6309e7a80f318da82ebf3f299eeb5d21da3/pkg/etcd/etcd.go#L84

I read the source code and found that the `CDCEtcdClient` just wraps `Client` as a field and adds more methods. Both of them were defined by us and the `Client` only are used for `CDCEtcdClient`.

So, my question is: can we merge them into a single type. For example, we delete the type `CDCEtcdClient` and put it's methods into the `Client` type.

**Comment by lonng** [comment]
@asddongmen I found the `CDCEtcdClient` was introduced by you, how do you think about that?

**Comment by asddongmen** [comment]
@lonng 
I did not introduce `CDCEtcdClient`, I just moved it from another package to here.
I think that `Client` is a simple warp for etcd's `clientV3.Client`, all methods' signatures of it are almost the same as `clientV3.Client`. The main purpose of it is to make some method of `clientV3.Client` more robust to use, so `Client` represents a `clientV3.Client` in CDC.
But `CDCEtcdClient `  add a bunch of methods that `clientV3.Client` didn't have, it is used by CDC only. 
So, maybe it is better to remain them separately.
